javascript/javascript1/week2/candyHelper.js

```diff
@@ -0,0 +1,36 @@
+//Candy Helper
+const boughtCandyPrices = [];
+const candyPrices = {
+  sweet: 0.5,
+  chocolate: 0.7,
+  toffee: 1.1,
+  "chewing-gum": 0.03,
+};
+function addCandy(candyType, weight) {
+  const pricePerGram = candyPrices[candyType];
+  if (pricePerGram) {
+    const totalPrice = pricePerGram * weight;
+    boughtCandyPrices.push(totalPrice);
+  } else {
+    console.log("Invalid candy type");
+  }
+}
+const amountToSpend = Math.random() * 100;
+function canBuyMoreCandy() {
+  let totalSpent = 0; // total amount spent on buying candies
+  for (let i = 0; i < boughtCandyPrices.length; i++) {
+    totalSpent += boughtCandyPrices[i];
+  }
+  return totalSpent < amountToSpend;
+}
+if (canBuyMoreCandy()) {
+  console.log("You can buy more, so please do!!!");
+} else {
+  console.log("Enough candy for you!!!");
+}
```

Comment:
I'd recommend that you have this logic inside of the `canBuyMoreCandy` function and return the messages that you now `console.log`.

Reply:
Done..